### ISSUE TITLE
Desk research for business case

### ISSUE THREAD

**Issue by holanita** [Green-Software-Foundation/state-of-green-software#48]
What: finalize the desk research for the business case
Who: Ismael
When: Nov 9 (delayed)

**Comment by holanita** [comment]
This ticket is delayed, waiting for Ismael's updates.

@Leamsi9 I have created tickets for each section (see mentioned above) so it's easier to track the status of each. From now on, can you add updates to these individual tickets to make it easier to track progress. We can then use these tickets to discuss the actual content with Asim so we have everything in one place. Hope this will work. Many thanks!

**Comment by holanita** [comment]
It turned out that Ismael will not be available today to give us updates or attend the meeting as per his Slack message. He should have some news for us tomorrow morning.

**Comment by Leamsi9** [comment]
Apologies for a chain of unexpected disruptions to my communications last week that interrupted my updates!

Desk research for the Business Case (pre-interview and survey data) is now complete. Currently extracting the headline data and data format for website. Should have website-ready v0.1 by tomorrow am.

**Comment by holanita** [comment]
OK no worries, looking forward to V0.1!

**Comment by holanita** [comment]
Adding @Leamsi9 's [latest Slack message](https://greensoftwarefdn.slack.com/archives/C0474E5KCTH/p1669630690260999) to make it easier to track and review:

Here is a first draft of the Business Case which can be used as a template, although it would be good to get your view on it and maybe jump on a call? I have tried to keep it short on text and visually focused as we discussed, with illustrative graphs and charts which the design team can recreate in a brand appropriate way.
https://docs.google.com/document/d/1cgy-38nOnW3L98d7mr4OnInHVxKpR2zd/edit?usp=sharing&ouid=102759123097087113446&rtpof=true&sd=true

**Comment by holanita** [comment]
Just adding a note here that this is 19 days behind schedule.

**Comment by holanita** [comment]
@Leamsi9 see below what @jawache has asked for in Slack. The conversation should continue here in this issue.

- [ ] review and respond to Asim's comments in the [google document](https://docs.google.com/document/d/1cgy-38nOnW3L98d7mr4OnInHVxKpR2zd/edit)
- [ ] put the headings, subheadings back in the document (I've added the table of contents for you to speed things up)
- [ ] add some content to explain more context of what will be in each section

The same document will be used for all other sections as well as Asim prefers to have a one pot wonder. Remember that this ticket is about the business case. You can discuss the details here or in comments in the doc. Next up is [knowledge](https://github.com/Green-Software-Foundation/state-of-green-software/issues/49) / TOC.

I'd say that the deadline for this is kind of asap / right now as we are too much behind schedule. What's your ETA?

**Comment by jawache** [comment]
I always imagined the business case to be a summary section. The other research _leads into_ the business case. So the business case for green software is something we conclude from our other research in the report, insights from the survey etc...

It's a very important section but that doesn't mean it needs to be completed first, in fact I would say for this to be tied into the rest of the report it should be one of the last sections to finish off.

Since it's not one of the commonly formatted ToC sections it's of less use in the design phase, so it's not a priority for Osama.

**Comment by holanita** [comment]
@jawache okay noted. what would you call the text that @Leamsi9 had already written? he calls it the _business case_.

**Comment by Leamsi9** [comment]
I have incorporated the business case section into: https://docs.google.com/document/d/1AhkCZWPcdFJBV_nk27xhip-2aiW-RBmb/

Will respond and incorporate Asim's comments directly in the document, not here.

**Comment by holanita** [comment]
OK. @Leamsi9 @jawache please keep the conversation going in comments so we can close this and move on to the next ones asap.

**Comment by holanita** [comment]
ETA of the rewrite: _Dec 16_ as per Ismael

**Comment by holanita** [comment]
@Leamsi9 let me simplify things a little bit. Hopefully this helps. I'm closing this task, will only have one task where I'll summarize all the essential details briefly.

So for the business case:
- this is supposed to be a summary section
- no urgency from a design perspective, can be the last one to complete
- [this](https://docs.google.com/document/d/1AhkCZWPcdFJBV_nk27xhip-2aiW-RBmb/edit) is the latest document that now contains the business case (see in section 2)
